web/components/inputs/FcSearchBarRequests.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <v-text-field
+    v-model="internalValue.query"
+    append-icon="mdi-magnify"
+    class="fc-search-bar-requests flex-grow-0 flex-shrink-0"
+    dense
+    hide-details
+    label="Search"
+    outlined>
+    <template v-slot:prepend>
+      <v-select
+        v-model="internalValue.column"
+        class="fc-search-bar-requests-column font-weight-regular mt-0 pt-0 title"
+        dense
+        hide-details
+        :items="itemsColumn"
+        outlined>
+      </v-select>
+    </template>
+  </v-text-field>
+</template>
+
+<script>
+import FcMixinVModelProxy from '@/web/mixins/FcMixinVModelProxy';
+
+export default {
+  name: 'FcSearchBarRequests',
+  mixins: [FcMixinVModelProxy(Object)],
+  props: {
+    columns: Array,
+    searchKeys: Object,
+  },
+  computed: {
+    itemsColumn() {
+      const searchableColumns = this.columns.filter(
+        column => Object.prototype.hasOwnProperty.call(this.searchKeys, column.value),
+      );
+      return [
+        { text: 'All Columns', value: null },
+        ...searchableColumns,
+      ];
+    },
+  },
+};
+</script>
+
+<style lang="scss">
+.fc-search-bar-requests {
```

Comment:
can we take one more stab at dealing with this?  it looks like the `fieldset` manages the border here, which kinda makes sense if you think about `<v-input>` as a container for one or more closely related input controls.